src/model/users.rs

```diff
@@ -158,6 +157,19 @@ pub struct Presence {
 	pub traveling_to_world: crate::id::OfflineOrPrivateOr<crate::id::World>,
 	/// The world that the user is in
 	pub world: crate::id::OfflineOr<String>,
+	/// If the user is rejoining the instance
+	#[serde(default)]
+	pub is_rejoining: serde_json::Value,
```

Comment:
Never seen this, but I guess I'll trust you on it possibly existing if I checked the API whilst playing...